src/hocs/ec-with-sorting/ec-with-sorting.js

```diff
@@ -13,40 +13,41 @@ const withSorting = createHOCc({
       type: Array,
       default: () => [],
     },
+    defaultSortCycle: {
+      type: Array,
+      default: () => [null, SortDirection.ASC, SortDirection.DESC],
```

Comment:
I think null should be always first and I wouldn't make its position configurable. I cannot imagine passing something like
`[SortDirection.ASC, null, SortDirection.DESC]`.

Reply:
Yep makes sense

- [ ] remove 'null' position